roles/openshift_cluster_monitoring_operator/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-openshift_cluster_monitoring_operator_image: quay.io/coreos/cluster-monitoring-operator:198d25d6
-openshift_cluster_monitoring_operator_prometheus_operator_repo: quay.io/coreos/prometheus-operator-dev
+openshift_cluster_monitoring_operator_image: quay.io/coreos/cluster-monitoring-operator:f0ca8a72
```

Comment:
why isn't this a semantic version tag(v3.10) like we use for other infrastructure images?  will it be moved to be one at some point?

Reply:
We are using the commit sha just for now. We will be moving to semantic version tags. @bparees is this a valid interim state?

Reply:
not my call, just found the choice interesting.  good to know it is not the long term plan, anyway.